en/docs/install-and-setup/setup/api-controller/advanced-topics/configuring-environment-specific-parameters.md

```diff
@@ -75,6 +75,57 @@ When there are multiple artifacts which needs to be added as deployment-related
         </tbody>
     </table>
 
+
+## Bundling the generated directory before import
+
+After generating the deployment directory, API controller is packed with a bundle command which provides the support
+to archive the directory without the need of external dependencies. This command will generate a `.zip` archive
+file of a given directory. If api_meta.yaml file ( or api_product_meta.yaml or application_meta.yaml) is included in
+the project, the created archive file name will be the combination of project name, version and the revision number.
+
+-   **Command**
+        ``` bash
+        apictl bundle -s <path-to-source-directory> 
+        ```
+        ``` bash
+        apictl bundle -s <path-to-source-directory>  -d <path-to-the-archive-destination>
+        ```
+        ``` bash
+        apictl bundle --source <path-to-source-directory> 
+        ```
+        ``` bash
+        apictl bundle --source <path-to-source-directory>  --destination <path-to-the-archive-destination>
+        ```
+        
+    !!! info
+            **Flags:**  
+            
+            -   Required :  
+                `--source` or `-s` : File path of the source directory to archive.
+            -   Optional :  
+                `--destination` or `-d` : Path of the directory where the archive file should be generated.    
```

Comment:
Here as well, Shall we remove the full stop to make the flag descriptions more consistent with other places?

Reply:
Fixed with b6dd23c33d3648b61e083aee1aee67878bebfb1d